vulcanai2/models/metrics.py

```diff
@@ -78,6 +80,8 @@ def get_class(self, in_matrix):
         :param n_matrix: one-hot matrix
         :return: 1D Class array
         """
+        if isinstance(in_matrix, torch.Tensor):
+            in_matrix = in_matrix.detach().numpy()
```

Comment:
Getting an error at this point after commenting out `model1.runtest` in the example
```
Traceback (most recent call last):
  File "fashion_conv_dense_test.py", line 136, in <module>
    model1.forward_pass(val_loader, convert_to_class=True),
  File "c:\work\aifred\vulcan2\master\vulcan2\vulcanai2\models\basenetwork.py", line 473, in forward_pass
    self.metrics.get_class(in_matrix=predictions)).float()
  File "c:\work\aifred\vulcan2\master\vulcan2\vulcanai2\models\metrics.py", line 84, in get_class
    in_matrix = in_matrix.detach().numpy()
TypeError: can't convert CUDA tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```
Same CUDA issue.

Reply:
Actually change it to: `in_matrix = in_matrix.cpu().detach().numpy()`

Reply:
im just gonna change it before it gets called into get_class